src/rust/collections/shared_ring.rs

```diff
@@ -110,6 +111,7 @@ mod test {
 
     /// Tests if we succeed to perform concurrent accesses to a shared ring buffer..
     #[test]
+    #[ignore]
```

Comment:
May we schedule a call with the others to discuss this?

Reply:
Call scheduled.

Reply:
We had the discussion and decided to leave this in this PR.

Reply:
We apparently didn't all remember the discussion, so I removed this from this PR.